apps/hashdotai/integrations/integration-list.json

```diff
@@ -0,0 +1,142 @@
+[
+  {
+    "name": "Linear",
```

Comment:
Appreciate this is a temp solution, but I think it would be easiest to have each integration in its own JSON file. We can defer that for now. I just think it's gonna become v cumbersome to add more than even a few of these in current form. Probably not worth addressing prior to just storing this data in HASH, though.

Reply:
I would expect that we are not adding many more before moving the data into HASH

Reply:
I think we'll want at least a ~dozen listed for when we switch over to this site (our next post-board milestone), so it depends when we envisage being able to store this info in HASH/what's actually blocking that. Just stable data? Also CDTs (enums)?

Reply:
Just stable data, probably. Most of the new ones will be ones to vote on, no? Which are small JSON objects which I think are probably easier to add to a single file than managing multiple, if anything. We can change it later though if not.